docs/manage-default-themes-and-styles.md

```diff
@@ -0,0 +1,157 @@
+
+[View Demo](https://surveyjs.io/form-library/examples/create-custom-ui-theme/ (linkStyle))
+
+### Custom CSS Classes
+
+If you want to apply custom CSS classes to all survey elements of a specific type, define a JavaScript object in which keys specify survey elements and values specify CSS classes. For information on the object structure, refer to the following file on GitHub: [defaultV2Css.ts](https://github.com/surveyjs/survey-library/blob/master/src/defaultCss/defaultV2Css.ts#L13). Assign this object to [`SurveyModel`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model)'s `css` property.
+
+[View Demo](https://surveyjs.io/form-library/examples/survey-customcss/ (linkStyle))
+
+In addition, the `SurveyModel` object raises events that allow you to override CSS classes for individual questions, panels, pages, and choice items. Refer to event descriptions in the API reference for more information:
+
+- [`onUpdateQuestionCssClasses`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onUpdateQuestionCssClasses)
+- [`onUpdatePanelCssClasses`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onUpdatePanelCssClasses)
+- [`onUpdatePageCssClasses`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onUpdatePageCssClasses)
+- [`onUpdateChoiceItemCss`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onUpdateChoiceItemCss)
+
+[View Demo](https://surveyjs.io/form-library/examples/survey-cssclasses/ (linkStyle))
+
+## Switch Between Themes
+
+If you add more than one SurveyJS theme to your application, call the `applyTheme(themeName)` method to specify which of the themes should be applied. Pass one of the following theme names as the method's argument:
+
+- `"defaultV2"`
+- `"modern"`
+
+The following theme names are also supported, but they are obsolete:
+
+- `"default"`
+- `"orange"`
+- `"darkblue"`
+- `"darkrose"`
+- `"stone"`
+- `"winter"`
+- `"winterstone"`
+
+The code example below shows how to apply the Default V2 theme:
+
+```js
+import { StylesManager } from 'survey-core';
+
+StylesManager.applyTheme("defaultV2");
+```
+
+> Previously, you needed to call the `applyTheme(themeName)` method even if your application includes only one SurveyJS theme. Since v1.9.69, a theme applies automatically if it is the only SurveyJS theme in the application.
+
+## Bootstrap Support (Obsolete)
+
+SurveyJS includes themes designed for use with Bootstrap and Bootstrap Material. However, they are considered obsolete and no longer receive updates.
+
+Previously, Bootstrap integration was part of the main package. Since v1.9.69, it is shipped as a separate module/script. The following code shows how to apply the Bootstrap or Bootstrap Material theme by importing a module:
```

Comment:
```suggestion
Previously, Bootstrap integrations were part of the main package. Since v1.9.69, they are shipped as separate modules/scripts. The following code shows how to apply the Bootstrap or Bootstrap Material theme by importing a module:
```

It is a module when it is added to a modular application, and a script when it is added to a monolithic application. You can see code examples for these two cases below.

Reply:
Thank you for the clarification. And I like your new version.